andela/hc-logros-infinitos
# #153728018 Add blog functionality

#### What does this PR do?
Users should be able to write blog posts on the health checks site

#### Description of Task to be completed?
Users should be able to write blog posts on the health checks site
Users should be able to share blog posts publicly

#### How should this be manually tested?
GIVEN users want to contribute knowledge
WHEN they login to their account
THEN they should be able to create categories, write blog posts and share those posts publicly

#### What are the relevant pivotal tracker stories?
[#153728018](https://www.pivotaltracker.com/story/show/153728018)
